includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -629,6 +629,12 @@ public function register_scripts() {
 		);
 
 		wp_set_script_translations( 'WCPAY_CHECKOUT', 'woocommerce-payments' );
+
+		if ( ! WC()->cart->needs_payment() && is_checkout() && ! has_block( 'woocommerce/checkout' ) ) {
+			WC_Payments::get_gateway()->tokenization_script();
```

Comment:
We may just directly `$this->tokenization_script()` as it's basically the same one but without calling an outside function to get another inside function.